code/modules/projectiles/projectile.dm

```diff
@@ -468,19 +468,18 @@
 	for(var/i in 1 to SSprojectiles.global_iterations_per_move)
 		if(QDELETED(src))
 			return
-		trajectory.increment(trajectory_multiplier)
-		var/turf/T = trajectory.return_turf()
-		if(T.z != loc.z)
+		//trajectory.increment(trajectory_multiplier)
+		//var/turf/T = trajectory.return_turf()
+		degstep(src, original_angle, 1)
+		hitscan_last = loc
+/* 		if(T?.z != loc?.z)

```

Comment:
More

Reply:
For this one it kind of removes previous functionality, projectiles won’t be able to cross Z levels